internals/secrethub/rm_test.go

```diff
@@ -0,0 +1,270 @@
+package secrethub
+
+import (
+	"bytes"
+	"errors"
+	"testing"
+
+	"github.com/secrethub/secrethub-go/internals/assert"
+
+	"github.com/secrethub/secrethub-cli/internals/cli/ui/fakeui"
+	"github.com/secrethub/secrethub-go/internals/api"
+	"github.com/secrethub/secrethub-go/pkg/secrethub"
+	"github.com/secrethub/secrethub-go/pkg/secrethub/fakeclient"
+)
+
+func TestRmCommand_Run(t *testing.T) {
+
+	var testErr = errors.New("test")
+
+	cases := map[string]struct {
+		cmd                RmCommand
+		in                 string
+		argPath            api.Path
+		promptOut          string
+		out                string
+		err                error
+		promptError        error
+		deleteSecretError  error
+		deleteVersionError error
+		deleteDirError     error
+		newClientError     error
+		getTreeError       error
+		getSecretError     error
+	}{
+		"success-force-dir": {
+			cmd: RmCommand{
+				force:     true,
+				recursive: true,
+				path:      "namespace/repo/dir",
+			},
+			argPath: "namespace/repo/dir",
+			out:     "Removal complete! The directory namespace/repo/dir has been permanently removed.\n",
+		},
+		"success-non-force-dir": {
+			cmd: RmCommand{
+				force:     false,
+				recursive: true,
+				path:      "namespace/repo/dir",
+			},
+			argPath: "namespace/repo/dir",
+			promptOut: "[WARNING] This action cannot be undone. This will permanently remove the namespace/repo/dir directory and all the directories and secrets it contains. " +
+				"Please type in the name of the directory to confirm: ",
+			in:  "namespace/repo/dir",
+			out: "Removal complete! The directory namespace/repo/dir has been permanently removed.\n",
+		},
+
```

Comment:
This and all other blank lines between test cases should be removed.

Reply:
This should now be solved! :)